src/odemis/driver/test/tescan_test.py

```diff
@@ -299,10 +484,12 @@ def test_move_abs(self):
         time.sleep(6)
         testing.assert_pos_almost_equal(self.stage.position.value, p)
 
+    # @skip("not working with the newer simulator Tescan Essence v1.2.2")
     def test_stop(self):
         """
         Check it's possible to stop the stage while it's moving.
         """
+        # TODO: make this method work again with Tescan Essence v1.2.2.
```

Comment:
Let's drop this TODO if this now works?

Reply:
I have created a task for this in the sw maintenance board to address this.
https://delmic.atlassian.net/browse/SWM-89?atlOrigin=eyJpIjoiYTM0Y2RhMzg0NWE4NGNjMjk4NDMxZTdlNTRkOTBjNTciLCJwIjoiaiJ9